frontend/src/ui/components/organisms/avatar-tile/AvatarTile.tsx

```diff
@@ -12,11 +13,33 @@ interface AvatarTileProps {
 
 export default function AvatarTile({
   firstName,
+  lastName,
   isBirthday,
   isCaptain,
   profilePictureUrl,
   onClick,
 }: AvatarTileProps): JSX.Element {
+  const getFirstLastName = (lastName: string) => {
```

Comment:
For me this code its so complex to read it, Refactor this and do the code more readable.

-How? You can move to helper, split it in small functions...
-Create test cases for all cases